src/main/java/de/dataelementhub/model/handler/importhandler/ImportHandler.java

```diff
@@ -0,0 +1,248 @@
+package de.dataelementhub.model.handler.importhandler;
+
+import static de.dataelementhub.dal.jooq.Tables.IMPORT;
+import static de.dataelementhub.dal.jooq.Tables.SCOPED_IDENTIFIER;
+import static de.dataelementhub.dal.jooq.Tables.STAGING;
+import static org.jooq.impl.DSL.count;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.dataelementhub.dal.jooq.enums.ElementType;
+import de.dataelementhub.dal.jooq.enums.ProcessStatus;
+import de.dataelementhub.dal.jooq.tables.pojos.ScopedIdentifier;
+import de.dataelementhub.dal.jooq.tables.records.ImportRecord;
+import de.dataelementhub.model.dto.element.StagedElement;
+import de.dataelementhub.model.dto.element.section.Member;
+import de.dataelementhub.model.dto.importdto.ImportDto;
+import de.dataelementhub.model.dto.importdto.ImportInfo;
+import de.dataelementhub.model.handler.element.NamespaceHandler;
+import de.dataelementhub.model.handler.element.section.IdentificationHandler;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.xml.XMLConstants;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import net.lingala.zip4j.ZipFile;
+import net.lingala.zip4j.exception.ZipException;
+import org.eclipse.persistence.jaxb.JAXBContextProperties;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.jooq.CloseableDSLContext;
+import org.jooq.Record2;
+import org.jooq.impl.SQLDataType;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.springframework.web.multipart.MultipartFile;
+
+public class ImportHandler {
+
+  /** Unzip received file and handle importing its content. */
+  public static void importFiles(
+      CloseableDSLContext ctx, List<MultipartFile> files,
+      String importDirectory, int userId,
+      int importId, String timestamp) {
+    String destination = importDirectory + "/" + userId + "/" + timestamp;
```

Comment:
Don't know if we need to support windows (@vabishaa ?), but if we do, this won't work. I think in both cases it would be better to use [File.separator](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#separator)

Reply:
I tested both on windows and they both work. The slash is replaced now by File.seperator